### PR TITLE
Feature/68 sign in page

### LINKED ISSUE #41: Integrating google authentication with Vue.js

### PR DESCRIPTION
**Pull request by tedspare**
# Summary

- Fix sign in/out button UX
- Use JWT rather than storing tokens directly in local storage

## Test Plan

- Sign in and out in multiple browsers and sessions

## Related Issues

Closes #41

### CODE REVIEW

File: backend/app/main/views.py [hunk 1]
```diff
@@ -106,25 +119,38 @@ def get_all_clients():
...
+    list_of_clients = []
+    for r in response_json["records"]:
+        name = r["fields"].get("Name")
+        notes = r["fields"].get("Notes")
+        email = r["fields"].get("Client Email")
+        attachments = r["fields"].get("Attachments")
+        if name is not None:
+            m = Client(name=name, email=email, notes=notes,
+                       attachments=attachments)
+            list_of_clients.append(m.serialize())
```
Comment (kraglalbert) [inline backend/app/main/views.py:131]: Why is this being added?

File: backend/app/main/views.py [hunk 2]
```diff
@@ -203,14 +233,17 @@ def handleEmailResponse(res):
         return errorMsg
 
 # SECURITY WARNING: Susceptible to DDoS Attack
+
+
```
Comment (kraglalbert) [inline backend/app/main/views.py:237]: nit: remove these extra lines

File: backend/app/main/views.py [hunk 3]
```diff
@@ -272,3 +296,74 @@ def send_mail():
...
+            id_number=user_info["id"]
+        )
+
+        # set the JWT access cookie in the response
+        resp = jsonify({"user": user.serialize()})
+        set_access_cookies(resp, access_token)
+
+        return resp, 200
+
+    return "Invalid id", 400
```
Comment (kraglalbert) [inline backend/app/main/views.py:361]: This overall seems a bit janky to me, take a look at https://github.com/kraglalbert/budgie-web/blob/master/backend/app/auth/views.py#L50 and the `refresh()`method in https://flask-jwt-extended.readthedocs.io/en/stable/tokens_in_cookies/ for the library's way of doing it

Comment (tedspare) [reply backend/app/main/views.py:361]: **TODO**: add JWT decorators

**Pull request merged**